extensions/rest/src/main/java/org/radargun/traits/RESTOperations.java

```diff
@@ -0,0 +1,22 @@
+package org.radargun.traits;
+
+import java.util.List;
+import javax.ws.rs.core.Cookie;
+import org.radargun.Operation;
+import org.radargun.WrappedHttpResponse;
+
+/**
+ * @author Martin Gencur
+ */
+@Trait(doc = "Http operations.")
+public interface RESTOperations {
+   String TRAIT = RESTOperations.class.getSimpleName();
+
+   Operation GET = Operation.register(TRAIT + ".Get");
+
+   RESTOperationInvoker getRESTInvoker();
+
+   interface RESTOperationInvoker {
```

Comment:
We do not want to change interfaces all the time, though there is (and probably will be) only one implementation, there will be multiple usages.
You wouldn't ignore that parameter - you would simply append that to the base URL. You should add the parameter to the test stage and keep it empty if you don't want to use it.